Gio-2.0.gir

```diff
@@ -18276,7 +18316,7 @@ This signal corresponds to the
           </parameter>
         </parameters>
       </glib:signal>
-      <glib:signal name="g-signal" when="last" version="2.26">
+      <glib:signal name="g-signal" when="last" detailed="1" version="2.26">
```

Comment:
Not sure how to handle this from bindings perspective as it used to not take a signal details, maybe we need a manual version that doesn't for < 2.72 which doesn't take a signal detail param?

Reply:
Yes